packages/pie-toolbox/src/code/drag/placeholder.jsx

```diff
@@ -19,11 +19,21 @@ export const PlaceHolder = (props) => {
   if (grid && grid.columns) {
     style.gridTemplateColumns = `repeat(${grid.columns}, 1fr)`;
   }
+  
   if (grid && grid.rows) {
     const repeatValue = grid.rowsRepeatValue || '1fr';
 
     style.gridTemplateRows = `repeat(${grid.rows}, ${repeatValue})`;
   }
+  
+  // The "type" is only sent through placement-ordering / placeholder
+  // It can be "choice" or "target"
+  // We apply a different style for the "choice" type
+  // For any other type, use a dashed black border and a white fill
+  if (type === 'choice') {
+    style.border = '1px solid #D1D1D1';
+    style.background = '#eeeeee';
```

Comment:
let's use https://github.com/pie-framework/pie-lib/blob/develop/packages/pie-toolbox/src/code/render-ui/color.js#L26 for consistency (PD-3944)